lib/net/ssh/connection/channel.rb

```diff
@@ -677,6 +678,16 @@ def forward_local_env(env_variable_patterns)
             end
           end
         end
+
+        # Set a +Hash+ of environment variables in the remote process' environment.
+        #
+        #   channel.set_remote_env foo: 'bar', baz: 'whale'
+        #
+        # rubocop:disable Naming/AccessorMethodName
```

Comment:
pls just disable Naming/AccessorMethodName from config

Reply:
Got it. Meanwhile, I'm trying to add some integration test for what I've done, and upgrade OpenSSH used in integration test from 7.4p1 to 7.9p1. Because I saw the codecov/patch check was not passed, seemed the test coverage was not enough. Will let you have a look after I finish.

Reply:
It's complining that `set_remote_env` in channel.rb is not tested you might get away without integation test but testing that line, but sure an integration would be great!